packages/core/src/language-server/binding.ts

```diff
@@ -141,8 +123,8 @@ export function bindLanguageServerPool({
           kind,
           range,
           diagnostics,
-          formating,
```

Comment:
Were these old apis? Or were whey wrong before?

They didn't seem to be 'our' properties, as i couldn't find them, but maybe i just missed them

Reply:
Not totally sure what you're asking, but these formatting and preference hashes are read from the VSCode configuration for the built-in TS/JS language server, but once you disable that (for "takeover" mode where Glint does everything) these hashes become empty, so I'm overriding with a hard-wired set of defaults.

Reply:
I just mean that it _looks_ sus that previously the key of `formatting` used and changing ut to all uppercase `FORMATTING` makes things work. Like, i assume then that `formatting` was always incorrect.

Reply:
Oh nevermind, i thought this was object shorthand, rather than positional argn

Reply:
(Discussed on Discord:) these are positional params, not an options hash.